### DISCUSSION TITLE
[Feature] Share photos of selected Person with other Immich users

### DISCUSSION THREAD

**Discussion by scorpio862** [immich-app/immich#4151] [Feature Request]
### The feature

I recently created an account for my daughter on my Immich server. 
I have thousands of images with her face but only few photos are in Albums which I can share with her. 
I wanted her to have an access to my photos with her face but I don't want to share all my photos using option "Add partner".

It would be great if there was an option to select a Person and share all the photos of this Person, with another Immich users. 
 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
This is considered "smart albums", which are a work in progress.

**Comment by jrasm91** [comment]
https://github.com/immich-app/immich/discussions/3306
